### ISSUE TITLE
[2.0][nightly] Long_running_horovod_tun_test is failing

### ISSUE THREAD

**Issue by scv119** [ray-project/ray#27182]
### What happened + What you expected to happen

2. Long_running_horovod_tun_test https://buildkite.com/ray-project/release-tests-branch/builds/832#0182422c-f590-4f0e-b05c-a64e2c2341ec

### Versions / Dependencies

releases/2.0.0

### Reproduction script

n/a

### Issue Severity

High: It blocks me from completing my task.

**Comment by krfricke** [comment]
Will be closed once https://github.com/ray-project/ray/pull/27169 is merged into master and cherry-picked onto the release branch
